kenya_compliance/kenya_compliance/overrides/server/stock_ledger_entry.py

```diff
@@ -224,7 +231,13 @@ def on_update(doc: Document, method: str | None = None) -> None:
         else:
             payload["sarTyCd"] = "11"
 
-    server_url = get_server_url(company_name, record.branch)
+
+    try:
+        server_url = get_server_url(company_name, record.branch) 
+    except Exception as e:
```

Comment:
Avoid catching the general Exception. Capture specific exceptions that you have a strategy to handle